### ISSUE TITLE
timeout without then or catch

### ISSUE THREAD

**Issue by hori-ryota** [azu/promise-test-helper#2]
Using mocha,
```javascript
it('without catch', function() {
  return shouldFulfilled(Promise.reject(new Error()));
});
```
and
```javascript
it('without catch', function() {
  return shouldRejected(Promise.reject(new Error()));
});
```
is timeout.

Return object isn't needed assertion, I have to create empty then method?

**Comment by azu** [comment]
@hori-ryota Thanks for report. I think its unexpected behavior. I'll fix it.

**Comment by azu** [comment]
I'v released [0.2.0](https://github.com/azu/promise-test-helper/releases/tag/0.2.0 "0.2.0").

These tests are passed, from now!

```javascript
it('without then', function() {
  return shouldFulfilled(Promise.resolve());
});
```

and

```javascript
it('without catch', function() {
  return shouldRejected(Promise.reject(new Error()));
});
```

**Comment by azu** [comment]
I'v released [0.2.0](https://github.com/azu/promise-test-helper/releases/tag/0.2.0 "0.2.0").

These tests are passed, from now!

```javascript
it('without then', function() {
  return shouldFulfilled(Promise.resolve());// => Passed
});
```

and

```javascript
it('without catch', function() {
  return shouldRejected(Promise.reject(new Error()));// => Passed
});
```

-----

On the other hand:

```js
it('without catch', function() {
  return shouldRejected(Promise.resolve());// => Failed
});
```

```js
it('without catch', function() {
  return shouldFulfilled(Promise.reject(new Error()));// => Timeout Error
});
```
